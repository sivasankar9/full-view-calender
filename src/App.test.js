// Copyright (c) 2019 Deere & Company

import AppComponent from './App';
import React from 'react';
import { shallow, mount, render } from './enzyme';

describe( 'Given the <appComponent/> component', () => {

    let appComponent;

    beforeEach( () => {

        appComponent = shallow(<AppComponent />);

    } );

    it( 'should match the snapshot', () => {

        expect( appComponent ).toMatchSnapshot();

    } );

} );
