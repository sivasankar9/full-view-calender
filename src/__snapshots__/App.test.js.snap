// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given the <ComponentFunctionCodeMaintainencePage/> component should match the snapshot 1`] = `undefined`;

exports[`Given the <appComponent/> component should match the snapshot 1`] = `ShallowWrapper {}`;
